tests/tests.toml

```diff
@@ -410,6 +410,10 @@ asr = true
 filename = "../integration_tests/test_builtin_oct.py"
 asr = true
 
+[[test]]
+filename = "../integration_tests/test_builtin_type.py"
+asr = true
+
```

Comment:
```suggestion
```

Tests this using `integration_tests/CMakeLists`

test: llvm, cpython

Reply:
Okay.

Reply:
The cpython test fails because the `type()` has a different return type there.